jxmlease/__init__.py

```diff
@@ -180,3 +183,26 @@ def __getattr__(self, name):
 # Now, import anything else we want.
 from .xmlparser import Parser, parse
 from .etreeparser import EtreeParser, parse_etree
+
+def emit_xml(obj, *args, **kwargs):
+    """Translate a Python dictionary or list to XML output.
+
+    This method internally creates an :py:class:`XMLDictNode` or
+    :py:class:`XMLListNode` object, as appropriate, and then calls the
+    :py:meth:`emit_xml <XMLNodeBase.emit_xml>` method of that class.
+
+    Any arguments you supply are passed on to the :py:meth:`emit_xml
+    <XMLNodeBase.emit_xml>` method of that class. Please see the
+    documentation for the class' :py:meth:`emit_xml <XMLNodeBase.emit_xml>`
+    method for information on arguments and return values.
+    """
+    try:
+        if isinstance(obj, (XMLDictNode, dict)):
+            parsed_obj = XMLDictNode(obj)
+        elif isinstance(obj, (XMLListNode, list, tuple)):
+            parsed_obj = XMLListNode(obj)
+        else:
+            raise TypeError
+    except Exception:
```

Comment:
This was purposeful. If there is *any* error converting these to the appropriate objects, they probably are not the correct type. But, I'm OK with changing it. Functionally, that will simply mean removing the try/except.